### PR TITLE
Add documentation for ImageEditor API

### PR DESCRIPTION
**Pull request by sgny**

### CODE REVIEW

File: reason-react-native/src/apis/ImageEditor.md [hunk 1]
```diff
@@ -1,45 +1,231 @@
...
+                ~width=displayWidth,
+                ~height=
+                  (
+                    if (state.imageLoaded) {
+                      1.;
+                    } else {
+                      0.;
+                    }
+                  )
+                  *. displayHeight,
```
Comment (cknitt) [inline reason-react-native/src/apis/ImageEditor.md]: Why not just put `displayHeight` instead of `1.` above?

Comment (sgny) [reply reason-react-native/src/apis/ImageEditor.md]: I was trying different multiples to align the text in the center of the box, but ultimately I had to pick `0.`, so `displayHeight` and `0.` would be cleaner.

Comment (sgny) [reply reason-react-native/src/apis/ImageEditor.md]: Removed that artifact

**Comment by MoOx** [comment]
Don't you think instead of those if/else we could use ternary thing `cond ? ifValue : elseValue` to make things more concise?

**Comment by sgny** [comment]
> Don't you think instead of those if/else we could use ternary thing `cond ? ifValue : elseValue` to make things more concise?

I was surprised to see `refmt` make that change, as I don't really like that syntax, I had only `if` statements.

Since the example isn't really for us, I did change the others as well.

**Pull request merged**